hack/uninstall-operator.sh

```diff
@@ -122,7 +122,11 @@ list_olm_resources() {
 }
 
 main() {
-	local operator="kepler-operator"
+	local operator=""
```

Comment:
lets rather make this a GLOBAL var and call a `init` function to initialize these globals which need to be calculated, after `parse_args`.